pkg/porter/lint_test.go

```diff
@@ -93,9 +93,11 @@ exec:
 	testcases := []struct {
 		format         string
 		wantOutputFile string
+		linterResults  *linter.Results
```

Comment:
What do you think if we keep this as a variable instead of a pointer? The reason why we want to keep this because if a test case modifies this variable value, it may cause subsequent tests be impacted with the modified value.

Reply:
Didn't thought about that side effect. Makes sense, I'll change it in a minute